00-RELEASENOTES

```diff
@@ -9,6 +9,44 @@ CRITICAL: There is a critical bug affecting MOST USERS. Upgrade ASAP.
 SECURITY: There are security fixes in the release.
 --------------------------------------------------------------------------------
 
+================================================================================
+Valkey 8.1.0 RC2  -  Released Thu 20 March 2025
+================================================================================
+Upgrade urgency LOW: This is the second release candidate of Valkey 8.1, with several bug fixes, 
+control on manual-failover timeout and expended module API to reduce module executed commands overhead.
+
+Performance/Efficiency Improvements - Core
+==========================================
+* Optimize bitcount command by SIMD (#1741)
+* Embed hash value in hash data type entries to reduce memory footprint (#1579)
+
+Cluster modifications
+====================
+* Add cluster-manual-failover-timeout configuration to control the timeout for manual failover (#1690)
+* Improve error massage reporting when invalid port is provided for cluster meet command. (#1686)
```

Comment:
Massage :massage: is nice way to handle errors, but I think you mean message. :envelope: